docker-compose-caddy.yaml

```diff
@@ -0,0 +1,416 @@
+version: "3.8"
```

Comment:
i don't think we need to store yet another copy of compose file in this repo. Let's just get it removed